source/brailleDisplayDrivers/nattiqbraille.py

```diff
@@ -0,0 +1,147 @@
+# brailleDisplayDrivers/nattiqbraille.py
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2011-2020

```

Comment:
As you say this is a new driver, you can change this as follows:

```suggestion
# Copyright (C) 2020 NV Access Limited, <your full name>

```

Reply:
ok, done.